### PR TITLE
Changed the level up chat messages for diggy 

### PR DESCRIPTION
**Pull request by SimonFlapse**
image

### CODE REVIEW

File: map_gen/Diggy/Feature/MarketExchange.lua [hunk 1]
```diff
@@ -138,25 +139,30 @@ local function update_market_contents(market)
...
 
     MarketExchange.update_gui()
 
     if (old_level < stone_tracker.current_level) then
+        if item_unlocked then
+            print(prefix..'We have reached level ' .. stone_tracker.current_level .. '! New items are available from the market!')
+        else
+            print(prefix..'We have reached level ' .. stone_tracker.current_level .. '!')
+        end
+        print('Mining Foreman: New wares at the market! Come get your ' .. name .. '!')
```
Comment (iltar) [inline map_gen/Diggy/Feature/MarketExchange.lua]: should probably be removed

Comment (SimonFlapse) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Yeah :)

File: map_gen/Diggy/Feature/MarketExchange.lua [hunk 2]
```diff
@@ -162,24 +162,23 @@ local function update_market_contents(market)
         else
             print(prefix..'We have reached level ' .. stone_tracker.current_level .. '!')
         end
-        print('Mining Foreman: New wares at the market! Come get your ' .. name .. '!')
         for _, buffs in pairs(config.buffs) do
             if (buffs.prototype.name == 'mining_speed') then
                 local value = buffs.prototype.value
-                print('Mining Foreman: Increased mining speed by ' .. value .. '%!')
+                if _DEBUG then print('Mining Foreman: Increased mining speed by ' .. value .. '%!') end
```
[the remark this answers is not in the document]
Comment (iltar) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Yeah, at some point we want to extract it

[the remark this answers is not in the document]
Comment (SimonFlapse) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Fixed in https://github.com/Refactorio/RedMew/pull/390/commits/cb0daa31bf1ff5f5b66dd798ed18b56bd8c8a1fe

**Pull request merged**